guides/common/modules/proc_renaming-server.adoc

```diff
@@ -70,10 +68,19 @@ For more information, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[
 ----
 # {hammer-smart-proxy} list
 ----
-. On {ProjectServer}, synchronize content for each {SmartProxyServer}:
++
+On {ProjectServer}, synchronize content for each {SmartProxyServer}:
```

Comment:
```suggestion
. On {ProjectServer}, synchronize content for each {SmartProxyServer}:
```

Is there a reason to _not_ use a ordered list item here?

Reply:
I didn't use the list item because I wanted to make the connection between the two steps a bit clearer: that the 2nd command (synchronizing content) needs to be done for each Capsule listed in the output of the 1st command. But yes, I can see now that it just looks weird. Good catch!